src/core/dfn.js

```diff
@@ -18,8 +29,24 @@ export function run(conf) {
     // Also, we should probably use weakmaps and weaksets here
     // to avoid leaks.
     const $dfn = $(dfn);
-    $dfn
-      .getDfnTitles({ isDefinition: true })
+    const dfnTitles = $dfn.getDfnTitles({ isDefinition: true });
+
+    // add automatic pluralization to `data-lt` attributes
+    // see https://github.com/w3c/respec/pull/1682
+    const normText = norm(dfn.textContent).toLowerCase();
+    if (
+      pluralizeDfn &&
+      !dfn.hasAttribute("data-lt-noPlural") &&
```

Comment:
this should be "data-lt-no-plural" (`noDefault` was badly named, but we shouldn't continue the bad practice)